base_ux/models/mail_template.py

```diff
@@ -0,0 +1,14 @@
+##############################################################################
+# For copyright and license notices, see __manifest__.py file in module root
+# directory
+##############################################################################
+from odoo import models, fields
+
+
+class MailTemplate(models.Model):
+    _inherit = 'mail.template'
+
+    active = fields.Boolean(
+        track_visibility='onchange',
```

Comment:
sacar

Reply:
removed, and removed from res_company too.